deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/codegen/OpenApiGeneratorCodeGenBase.java

```diff
@@ -1,8 +1,6 @@
 package io.quarkiverse.openapi.generator.deployment.codegen;
 
-import static io.quarkiverse.openapi.generator.deployment.SpecConfig.API_PKG_SUFFIX;
-import static io.quarkiverse.openapi.generator.deployment.SpecConfig.MODEL_PKG_SUFFIX;
-import static io.quarkiverse.openapi.generator.deployment.SpecConfig.getResolvedBasePackageProperty;
+import static io.quarkiverse.openapi.generator.deployment.SpecConfig.*;
```

Comment:
Avoid star imports

Reply:
IDE things... I'm using the quarkus code style, so this configuration is not defined there? I guess I have to manually fix it in the IDE. Sorry, new IDE on Windows. I'll revert it back. Since the formatted plugin didn't complain I thought it was ok.